src/components/Input/index.tsx

```diff
@@ -1,9 +1,10 @@
 import React, {
```

Comment:
Check this file, it has some warnings.

Reply:
Yes...
For now we will ignore them.

from useCallback
he complains, but it needs to be like this because this prop is passed in two moments, one by us when we set up the component input <input onChange={} />

another by the `react-hook-form` lib

which makes it dynamic for each person to have their independence, it needs to be like this.

esLint is good but dumb in some cases

another and useImperativeHandle
it needs to be re-called and this prop that ESlint complains about is there to do that
ESlint cannot identify it and then complains

But in both cases it doesn't generate behavior problems, they're just things that ESlint doesn't know how to deal with